sdk-python/livescoreapi.py

```diff
@@ -10,11 +10,50 @@
 
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
+    
+    Methods
+    -------
+    _get_html_file(link)
+    
+    get_all()
+    
+    """
+    def __init__(self,key,secret):
```

Comment:
Looks great. Part of writing nice structured code is leaving a space after a comma. For example:
`def __init__(self, key, secret):` it makes the code more readable